src/App.tsx

```diff
@@ -1,24 +1,160 @@
 /* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useState, useEffect, useMemo } from 'react';
 import { UserWarning } from './UserWarning';
+import { client } from './utils/fetchClient';
+import { getTodos, getPostTodos, deleteTodos } from './api/todos';
+import { Todo } from './types/Todo';
+import { NewTodo } from './types/NewTodo';
+import { TodoList } from './components/TodoList/TodoList';
+import { TodoHeader } from './components/TodoHeader/TodoHeader';
+import { TodoFooter } from './components/TodoFooter/TodoFooter';
+import { SortEnum } from './types/sort';
 
-const USER_ID = 0;
+const USER_ID = 10589;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [select, setSelect] = useState('all');
+  const [isError, setIsError] = useState(false);
+  const [querySearch, setQuerySearch] = useState('');
+
+  const lengTodos = todos.filter(todo => todo.completed === false).length;
+
+  const getTodosAll = async () => {
+    try {
+      const receivedTodos = await getTodos(USER_ID);
+
+      setTodos(receivedTodos);
+    } catch {
+      setIsError(true);
+    }
```

Comment:
I would show a certain error

```suggestion
    } catch (e) {
      setErrorMessage(e.message || 'Something went wrong.');
    }
```